source/includes/options-conf.yaml

```diff
@@ -707,19 +707,22 @@ description: |
    .. versionadded:: 3.4
 
    .. versionchanged:: 3.6
+      \ 
 
-      - Add support for :term:`zlib` compressor.
+   - Add support for :term:`zlib` compressor.
 
-      - Enabled by default. To disable, set to ``disabled``.
+   - Enabled by default using the :term:`snappy` compressor. To
+     disable compression, set this option to ``disabled``.
 
    Enables network compression for communication between this
    :program:`mongod` or :program:`mongos` instance and:
 
    - other members in the deployment, if a member of a replica set or a
      sharded cluster
 
-   - a :program:`mongo` shell
-   
+   - a :program:`mongo` shell when network compression is enabled
+     (disabled by default)
```

Comment:
the phrase "when network compression is enabled (disabled by default)":

-- unclear that you are referring to mongo shell's compression enablement
-- given that this is followed by a blurb that states that  both sides must have this feature enabled (whether other members of sharded cluster, etc), it's unnecessary as stated.  We can either just state that ``mongo`` shell disables this feature by default. or just remove it all together.